variables.tf

```diff
@@ -229,9 +324,89 @@ DESCRIPTION
   nullable    = false
 }
 
+variable "sku" {
+  type        = string
+  default     = "Standard"
+  description = "The SKU of the EventGrid Namespace (Standard or Premium)"
+
+  validation {
+    condition     = contains(["Standard", "Premium"], var.sku)
+    error_message = "The SKU must be either 'Standard' or 'Premium'."
+  }
+}
+
 # tflint-ignore: terraform_unused_declarations
```

Comment:
Either remove this comment or remove the variable if tags are not supported in these resource types. If tags are supported, please add them to the resource and sub resources.